<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>prn2printer</string>
  </property>
  <property name="statusTip">
   <string>wernix</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="chooseFileButton">
      <property name="text">
       <string>wybierz plik .PRN</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="filePath">
      <property name="text">
       <string>(wybierz plik)</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Ścieżka drukarki</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="printerPath"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="printButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>drukuj</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>chooseFileButton</tabstop>
  <tabstop>printerPath</tabstop>
  <tabstop>printButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
